lib/beaker-hostgenerator/data.rb

```diff
@@ -147,6 +147,15 @@ def osinfo
             'template' => 'amazon-7-arm64'
           }
         },
+        'archlinux2-64' => {
+          :general => {
+            'platform'           => 'archlinux-2-x64',
+            'packaging_platform' => 'archlinux-2-x64'
+          },
+          :vagrant => {
+            'box' => 'archlinux/archlinux',
```

Comment:
@shibumi can you have a look here and verify this is the correct/official vagrant image?

Reply:
https://app.vagrantup.com/archlinux at least lists itself as official

Reply:
Yes, that's the official image.